applications/ShapeOptimizationApplication/python_scripts/model_part_controller_factory.py

```diff
@@ -42,7 +42,8 @@ def __init__(self, model_settings, model):
             },
             "mesh_motion" : {
                 "apply_mesh_solver" : false
-            }
+            },
+            "iteration_directory": "optimization_iterations"
```

Comment:
I'm also in favor of removal of this setting and fixing it (We already have a fixed folder name "OptimizationResults", so this could be called "OptimizationIterations"). @armingeiser ?

Reply:
:+1: from my side

Reply:
done :)

Reply:
Did you think about renaming it to "DesignIterations"? I would personally find it more descriptive and it would also a bit more distinguishable from the "OptimizationResults" folder. But as I said, your call after all.

Reply:
done :)